.gitignore

```diff
@@ -12,6 +12,7 @@
 /Docs-v*.zip
 /Precompiled-v*.zip
 /.eunit
+/.idea
```

Comment:
It should be removed. Files that are related to editors and so on should go to a global gitignore in your machine, otherwise it is easy to see how this file would become huge with editor concerns. :) More info here: https://help.github.com/articles/ignoring-files/